src/shared/locked-status/use-check-lock-status.js

```diff
@@ -38,7 +38,17 @@ const isDataInputPeriodLocked = ({
     )
 }
 
-const isOrgUnitLocked = ({
+/**
+ * An org unit is locked not based on the current date,
+ * but based on the selected period.
+ *
+ * -> If org unit's close date is before the period's end date,
+ *    then the user is not allowed to modify data.
+ *
+ * -> If the org unit's open date is after the period's start date,
+ *    then the user is not allowed to modify data.
+ */
+const isOrgUnitTimeConstraintWithinDataInputPeriodConstraint = ({
```

Comment:
Why isn't `isOrgUnitLocked` good enough? This starts to be a bit too long imo. If there's it's used for other things other than lockstatus or it's actually other constraints that the lockstatus is based on its fine, but it's only for lockstatus, I think we should keep it. 
Since that describes what it's used for, not the implementation details of what makes it locked, which could in theory change, and you would need another function.